moveit_planners/ompl/ompl_interface/src/ompl_interface.cpp

```diff
@@ -202,13 +203,25 @@ bool ompl_interface::OMPLInterface::loadPlannerConfiguration(
   for (XmlRpc::XmlRpcValue::iterator it = xml_config.begin(); it != xml_config.end(); ++it)
   {
     if (it->second.getType() == XmlRpc::XmlRpcValue::TypeString)
+    {
       planner_config.config[it->first] = static_cast<std::string>(it->second);
+    }
     else if (it->second.getType() == XmlRpc::XmlRpcValue::TypeDouble)
-      planner_config.config[it->first] = boost::lexical_cast<std::string>(static_cast<double>(it->second));
+    {
+      // convert to string using no locale
+      std::ostringstream oss;
+      oss.imbue(std::locale());
```

Comment:
I'm not sure that's really neccessary at all.
If so, I guess you need to use [`std::locale::classic()`](https://en.cppreference.com/w/cpp/locale/locale/classic)

Reply:
Yes, using `std::locale::classic()` is indeed better. Omitting it would probably/usually/maybe also be right unless somebody somewhere set the global locale to something else.

As a side note, using `std::stod` for the inverse operation does not work as this uses the c locale internally which does seem to get set automatically from the system.